src/components/10-atoms/carousel/index.js

```diff
@@ -68,12 +68,30 @@ class AXACarousel extends InlineStyles {
   // private methods
 
   _getSlides(usedThis = this) {
-    // usedThis as a property to encapsulate for unit tests
-    const slots = usedThis // do not use this.shadowRoot because of IE
-      .querySelector('slot')
+    const slotsSelectedByShadowRootFiltered = usedThis.shadowRoot
+      .querySelector('.js-carousel__wrapper > slot')
```

Comment:
This has to be tested in IE. It probably works but we should make sure, because ShadyCss is messing with our structure.

Reply:
This line of code do not work in IE. But i have a if and a comment at line 87-89.